tests/Hosting/Environment/EnvironmentDecoratorMethodsTest.php

```diff
@@ -96,6 +96,7 @@ public function getEnvironmentInterfaceGetters()
             ['getDefaultHostName', 'test1-123.test.hosting.acquia.com'],
             ['getDatabaseClusterList', [1,2]],
             ['getDefaultDomainName', 'test1stg.acquia-sites.com'],
+            ['getDocumentRootPath', '/var/www/html/test.test/docroot'],
```

Comment:
meh, i don't think it matters for this test; the goal of the test is to validate that the decorator responds with the result of the wrapped method.